### PR TITLE
Remove the R_REJUVINATE flag and replace with R_POLL

### PR DESCRIPTION
**Pull request by optimumtact**
Rejuvenation lives under R_ADMIN and R_POLL is a new flag you can grant
to someone to allow them to create polls

the REJUV flag and define are still usable but simply grants and act like R_ADMIN instead and will be phased out eventually

### CODE REVIEW

**Comment by optimumtact** [comment]
@MrStonedOne will need admin ranks updated on disk

**Comment by Cyberboss** [comment]
Give this to coders tbh

**Comment by Really-Good-Soda-Flavor** [comment]
beautifully done tbh

File: code/__DEFINES/admin.dm [hunk 1]
```diff
@@ -30,7 +30,7 @@
 #define R_POSSESS		64
 #define R_PERMISSIONS	128
 #define R_STEALTH		256
-#define R_REJUVINATE	512
```
Comment (MrStonedOne) [inline code/__DEFINES/admin.dm]: You could keep this defined as 2 then your code supports legacy configs (if you keep the parser line, could even have it warn)

**Review by Cyberboss — CHANGES_REQUESTED** [review]
You need to change the lists in admin_ranks.dm to actually give people with the perm the verb

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Also update admin_ranks.txt

File: code/__DEFINES/admin.dm [hunk 2]
```diff
@@ -30,11 +30,16 @@
...
+#define R_POLL			512
 #define R_VAREDIT		1024
 #define R_SOUNDS		2048
 #define R_SPAWN			4096
 
+#if DM_VERSION > 512
+#error Remove the flag below , its been long enough
+#endif
+#define R_REJUVINATE	2 #legacy , remove post 512, it was replaced by R_POLL
```
Comment (MrStonedOne) [inline code/__DEFINES/admin.dm]: I don't think you can use comments in define lines.

File: code/__DEFINES/admin.dm (hunk 1 above)
Comment (optimumtact) [inline code/__DEFINES/admin.dm]: done

File: code/__DEFINES/admin.dm (hunk 2 above)
Comment (optimumtact) [inline code/__DEFINES/admin.dm]: reeeeeee

Comment (MrStonedOne) [inline code/__DEFINES/admin.dm]: also this wouldn't be a comment. that would be // or /**/

Comment (optimumtact) [inline code/__DEFINES/admin.dm]: too much python

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**